src/SearchParameters/index.js

```diff
@@ -443,6 +454,22 @@ SearchParameters.prototype = {
     } );
   },
   /**
+   * addTagRefinement adds a tag to the list used to filter the results
+   * @param {string} tag tag to be added
+   * @return {SearchParameters}
+   */
+  addTagRefinement : function addTagRefinement( tag ) {
+    if( this.tagFilters ) {
+      return this.setQueryParameter( "tagFilters", undefined )
```

Comment:
Re-thinking about that, I would rather go for an error/exception because it doesn't make sense to "loose" previous filters. Such "hidden" behavior are never understood correctly. What do you think?